### DISCUSSION TITLE
Clarifing .NET vs MAUI support

### DISCUSSION THREAD

**Discussion by axa88** [shinyorg/shiny#1339] [Q&A] [answered]
Hi. im new here.
Answer #1107 seems to clearly suggest support for MAUI, though the [readme](https://github.com/shinyorg/shiny/blob/master/readme.md#supports) seems to list supported platforms including:

    .NET 7 for Android
    .NET 7 for iOS
    .NET 7 for MacCatalyst
    .NET 7 for Windows (Core & BLE)

Where .NET for each platform support isn't technically support for MAUI is it?
Could this be clarified, is MAUI simply supported and im just confused?

**Comment by aritchie** [comment]
MAUI is a UI framework, not an operation system/platform.  Shiny is a framework for the device hardware.  The two can coexist on the platforms Shiny supports.

MAUI does offer some easy plug points that makes things easier to integrate, thus Shiny does have a hosting package to easily integrate with MAUI.

**Comment by axa88** [comment]
Ok. I have to apologize as what you stated is over my head.
I will reframe my question.

I understand MAUI is a UI framework and not an OS. Im coming to realize Shiny is a device hardware framework.
I want to use bluetooth LE with my MAUI projects which target, Android,  iOS and WinUI.
Will I be able to use the Shiny library to accomplish the work of abstracting the interaction with the Bluetooth hardware in my MAUI application?

**Reply by aritchie** [reply, marked as answer]
Yes, just not windows.  I haven't finished the implementation

**Reply by axa88** [reply]
Thanks for that clarification. Since id like WinUI support, shiny is probably not for me right now.
ETA on WinUI?
